### ISSUE TITLE
"logout button" doesn't work. 

### ISSUE THREAD

**Issue by bobdenotter** [bolt/Members#27]
See screenshot: 

![screen shot 2016-05-31 at 15 08 38](https://cloud.githubusercontent.com/assets/1833361/15675300/a50043f8-2741-11e6-9668-ef8218139403.png)

Either the _**form**_ should submit to `/authentication/logout/` or it should be an `<a href='/authentication/logout/'>`, i think.

**Comment by SahAssar** [comment]
IMO the form should submit a POST to `/authentication/logout`. GET's are not for actions. Will get on this soon.

**Comment by GawainLynch** [comment]
Neither, it should be, and needs to be both. You will break some implementations that exists already.

**Comment by bobdenotter** [comment]
> Neither, it should be, and needs to be both. 

Is that you, master Yoda? :-p

**Comment by GawainLynch** [comment]
Haha! Yeah, I did say I'd help those that took over :smile: 

OK, route needs to take GET & POST as you for certain reasons need to only rely on just hitting the route to logout, it's single function.

The buttons, I suck at HTML so go nuts… one of the big "debates" I kept having with "that" site @bobdenotter was consistency, as people want to put things like logout buttons places and they want to do it in Twig, and with the same blah blah blah as they use for the other blah blah blah … Totally happy for someone with proper HTML knowledge to make a call on that one! :+1:

**Comment by SahAssar** [comment]
Right, I mostly meant that the default should be a POST as that is (usually) not cached, while a GET might be. Sorta like the whole Steam caching debacle last Christmas.

**Comment by GawainLynch** [comment]
Oh right… my only concern was removing the GET… been a while since I looked at the logic on that route, but from memory it doesn't are about the request (and its contents) beyond hitting the controller route … so latter problem __should__ be mitigated. That should be reassessed though if someone has time! :smile:
